src/standalone/Snow/snow_parameterizations.jl

```diff
@@ -170,9 +170,11 @@ function snow_thermal_conductivity(
     parameters::SnowParameters{FT},
 ) where {FT}
     _κ_air = FT(LP.K_therm(parameters.earth_param_set))
+    _ρ_ice = FT(LP.ρ_cloud_ice(parameters.earth_param_set))
     κ_ice = parameters.κ_ice
     return _κ_air +
-           (FT(7.75e-5) * ρ_snow + FT(1.105e-6) * ρ_snow^2) * (κ_ice - _κ_air)
+           (FT(0.07) * (ρ_snow / _ρ_ice) + FT(0.93) * (ρ_snow / _ρ_ice)^2) *
```

Comment:
Are the coefficients here from Bonan's book? And do they change in this PR because the dimensionality of the thermal conductivity changed?

Reply:
The original coefficients were from Bonan's book, yes!  but it's not ideal to have coefficients with units  (the first would be per unit density, the second per unit density squared, but it might not be clear if these are kg/m^3 or something else for density). so I basically multiplied 7.75e-5 rho_snow by rho_ice/rho_ice, or 0.07 = 917*7.75e-5.

Also, it is clear now that if you plug in rho_snow = rho_ice, you get back k_air + (0.07+0.93)*(k_ice - k_air) = k_ice.

Reply:
I updated the doc string